Doc/library/sqlite3.rst

```diff
@@ -919,11 +919,20 @@ Connection objects
 
       .. versionadded:: 3.7
 
+.. _SQLite limit category: https://www.sqlite.org/c3ref/c_limit_attached.html
```

Comment:
Usually I've seen these listed at the end of the section where they are used, not before.  I don't think we have any convention about where to put them though, so if you think it's more readable putting it before, then I guess it's ok.

Reply:
I have no preference; I'm fine with having them at the end of sections.